enterprise/backend/test/metabase_enterprise/public_settings_test.clj

```diff
@@ -15,3 +17,30 @@
     (public-settings/enable-password-login! false)
     (is (= false
            (public-settings/enable-password-login)))))
+
+(deftest properties-token-features-test
```

Comment:
Not sure why this is test is in this namespace, given it's testing the API. 

I think it should go in `test/metabase/api/session_test.clj` near the other GET "/session/properties" tests https://github.com/metabase/metabase/blob/9515d70bceb3c600f23f9e5b4b8314028690b8bf/test/metabase/api/session_test.clj#L353

Reply:
Agree...in general EE tests (and code) could use a reorganization